src/Pages/ProjectDetails.jsx

```diff
@@ -29,7 +29,8 @@ export default function ProjectDetails() {
   const { country } = profile;
 
   const countDonations = (donations) => {
-    return donations.reduce(
+    const donationAmounts = donations.map((donation) => donation.amount);
+    return donationAmounts.reduce(
```

Comment:
Ah, Donation is an object. In that case you could use this reduce:

```
donations.reduce((a, b) => a + b.amount, 0)
```

`a` will be the running total, `b` is the current object, and the trailing `, 0` is the initial value of `a` at the start of the reduce

Reply:
Refactored, way cleaner approach, thank you!